src/modules/core/hooks/useGetParams.tsx

```diff
@@ -0,0 +1,24 @@
+import { useLocation } from 'react-router-dom';
```

Comment:
move this file to folder `vault/hooks` and exports on index

Reply:
This hook also return a function called `getPathParam`, that we can get any param from anywhere. 
It has this `vaultPageParams` because we're working in the vault provider but we can also create a `TransactionPageParams`, for exemple. 
So, that can be a good idea to keep it in core, cause can be helpful in any route.